tox.ini

```diff
@@ -19,3 +19,12 @@ usedevelop=true
 commands=
 	pytest --cov=cloudimg {posargs}
 	coveralls
+
+[testenv:cov]
+deps=
+	-rrequirements-test.txt
+	pytest-cov
+usedevelop=true
+commands=
+	pytest --cov-report=html --cov=pubtools {posargs}
```

Comment:
This doesn't look right - it's going to run the same test suite more than once.  Was that intentional?

Maybe one of them is just a mistake.  But if you really intended to get coverage for both "pubtools" and "cloudimg", I think you'd want to run pytest once, with two instances of the --cov argument, rather than running pytest twice.

Reply:
Now it looks broken as there are conflict resolution markers in the file (<<<<, ==== etc.)

Reply:
please check now.

Reply:
The content looks right, could you please rebase this into fewer commits?